<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DCalculadora</class>
 <widget class="QDialog" name="DCalculadora">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLCDNumber" name="lcdResultado"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <item row="7" column="0">
          <widget class="QPushButton" name="bUno">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="7" column="2">
          <widget class="QPushButton" name="bTres">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="5" column="2">
          <widget class="QPushButton" name="bNueve">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0" colspan="2">
          <widget class="QPushButton" name="bCero">
           <property name="text">
            <string>0</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QPushButton" name="bDos">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QPushButton" name="bSiete">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="8" column="2">
          <widget class="QPushButton" name="bComa">
           <property name="text">
            <string>,</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QPushButton" name="bOcho">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QPushButton" name="bCinco">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="bCuatro">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="6" column="2">
          <widget class="QPushButton" name="bSeis">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QPushButton" name="bIgual">
           <property name="text">
            <string>=</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bC">
           <property name="text">
            <string>c</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="bSuma">
           <property name="text">
            <string>+</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
